qemu/tests/watchdog.py

```diff
@@ -143,6 +143,10 @@ def check_guest_reboot(pattern):
             else:
                 f_param = "running"
 
+            # according to the performance issue, add 5sec before checking
+            # the vm status
+            time.sleep(5)
+
             if not utils_misc.wait_for(
                 lambda: vm.monitor.verify_status(f_param),
                     response_timeout, 0, 1):
```

Comment:
How about sleep in `utils_misc.wait_for`
```
def wait_for(func, timeout, first=0.0, step=1.0, text=None, ignore_errors=False):
```

Reply:
updated